docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,164 @@
+Mahti are considerably more powerful than the V100 GPUs on Puhti, which
+makes Mahti also suitable for machine learning applications. In contrast to Puhti,
+on Mahti only GPU nodes have fast local NVMe storage available.
+
+### LUMI
+
+LUMI is one of the fastest supercomputers in the world. It is intended primarily
+for running computations that benefit from the large amount of high-performance
+GPUs in its LUMI-G hardware partition. Whereas the GPUs on Puhti and Mahti are
+manufactured by Nvidia, the LUMI GPUs are made by AMD, so make sure that your
+GPU applications are able to run on AMD GPUs. LUMI has
+[its own documentation pages](https://docs.lumi-supercomputer.eu/).
+
+## Available interfaces
+
+### Web interface
+
+Puhti, Mahti and LUMI each have
+[their own web interface](../../computing/webinterface/index.md), which allows
+interacting with the supercomputer using a web browser. The web interface is a
+good choice for interactive computing, such as analyzing, exploring
+and visualizing data. For this purpose, the web interface features multiple
+interactive applications, like
+[Visual Studio Code](../../computing/webinterface/vscode.md),
+[Jupyter](../../computing/webinterface/jupyter.md) and
+[RStudio](../../computing/webinterface/rstudio.md), as well as an
+[accelerated visualization app](../../computing/webinterface/accelerated-visualization.md)
+for GPU-accelerated visualization and rendering. For demanding computation,
+like running full-scale simulations or training neural networks, you should use
+the command-line interface, since it allows you to access more resources and
+schedule your jobs.
+
+### Command-line interface
+
+While many of the interactive applications in the web interface, such as
+Jupyter and RStudio, are easy to use and thus a good starting point for using
+CSC supercomputers, their computing capacity is limited to relatively
+low-resource interactive usage. If you need access to more resources (e.g.
+multiple CPU nodes or GPUs) or if you require efficiency over interactivity,
+it is a good idea to switch to using the text-based command-line interface to
+interact directly with the supercomputer's
+[Linux operating system](./env-guide/index.md). While this way of working may
+seem archaic, it is truly powerful once you get used to it.
+
+The main benefit of using the CLI is getting to
+[submit your computations as batch jobs](../../computing/running/getting-started.md)
+to the SLURM job scheduler. Requesting a large amount of resources
+for an interactive application like Jupyter typically entails having to wait for
+several hours before finally logging in to manually run the job. This is
+both inconvenient and wasteful, as your interactive session may start at a time
+when you are unable to access your workstation, which means that several hours
+of potential runtime can go unused. When you submit a batch job to SLURM, your
+job is run automatically when the requested resources become available.
```

Comment:
Unsure about this paragraph. It is not quite convincing, but also do not have a better idea right now... 

Maybe rather than that you might get the resources when you are unable to access the workstation it could focus on that you really do not have to be on your workstation while the job is running, ie there does not need to be any connection between your computer and the supercomputer?

Somewhere here could also be mentioned that running batch job instead of interactively running stuff requires some adjustment, ie level of automation, which may seem tedious but in addition to "not having to care while it is running", also brings the benefit of making the job more reproducible?

Reply:
"The CLI allows you to submit your computations as batch jobs to the SLURM job scheduler, which runs them as soon as the requested resources are available. In practice, this means that getting work done does not necessitate being tied to your workstation. While setting up this automation can require some more planning on your part, in the long run it makes it more reproducible both to yourself and other parties, such as reviewers and collaborators."